privacy-snapshot.json

```diff
@@ -70,6 +70,7 @@
   "on-ramp-content.uat-api.cx.metamask.io",
   "phishing-detection.api.cx.metamask.io",
   "portfolio.metamask.io",
+  "app.metamask.io",
```

Comment:
Should we include both of these urls `portfolio.metamask.io` and `app.metamask.io` @MetaMask/extension-privacy-reviewers?